spec/pagseguro/installment/collection_spec.rb

```diff
@@ -0,0 +1,48 @@
+require "spec_helper"
+
+describe PagSeguro::Installment::Collection do
+  let(:installments) do
+    [
+      { amount: "100", card_brand: "visa" },
+      { amount: "110", card_brand: "visa" }
+    ]
+  end
+  subject { described_class.new({installments: installments}) }
+
+  describe "initialization" do
+    context "when options[:errors] is present" do
+      let(:errors) { double(:errors) }
+      subject { described_class.new({errors: errors}) }
+
+      it "sets errors" do
+        expect(subject.errors).to eq(errors)
+      end
+
+      it "has no installments" do
+        expect(subject.empty?).to be_truthy
```

Comment:
@dmitryrck you are right, I will create a task to change every spec like this one.